components/playlist/browser/resources/components/playlistInfo.tsx

```diff
@@ -46,16 +46,20 @@ export default function PlaylistInfo ({
 }: Props) {
   return (
     <PlaylistInfoContainer className={className}>
-      <PlaylistName>
+      <PlaylistName className='playlist-name'>
         {isDefaultPlaylist ? 'Play Later' : playlistName}{' '}
       </PlaylistName>
-      <PlaylistDetail>{itemCount} items</PlaylistDetail>
+      <PlaylistDetail className='playlist-detail'>
+        {itemCount} items
+      </PlaylistDetail>
       <>{!!totalDuration && <PlaylistDetail>{totalDuration}</PlaylistDetail>}</>
       <>
-        {(!!itemCount ||
-          false) /* TODO(sko) We can't get the file size for now */ && (
-          <PlaylistDetail>300 mb</PlaylistDetail>
-        )}
+        {
+          // TODO(sko) We can't get the file size for now
+          // !!itemCount && (
+          //   <PlaylistDetail className='playlist-detail'>300 mb</PlaylistDetail>
+          // )
+        }
```

Comment:
Temporarily commented out due to The Sonar cloud

Reply:
So annoying :laughing: We can probably get @bsclifton to help if we can't sort it out.